tools/expotools/src/Packages.ts

```diff
@@ -78,7 +78,7 @@ class Package {
     }
     const buildGradle = fs.readFileSync(path.join(this.path, this.androidSubdirectory, 'build.gradle'), 'utf8');
     const match = buildGradle.match(/^group ?= ?'([\w.]+)'\n/m);
-    return (match && match[1]) ? match[1] : null;
+    return match?.[1];
```

Comment:
This will return undefined if `match` is null. TypeScript should have detected this.

```suggestion
    return match?.[1] ?? null;
```